Experiments/Experiments/DefaultFeatureFlagService.swift

```diff
@@ -39,6 +39,13 @@ public struct DefaultFeatureFlagService: FeatureFlagService {
             return buildConfig == .localDeveloper || buildConfig == .alpha
         case .wpcomSignup:
             return buildConfig == .localDeveloper || buildConfig == .alpha
+        case .performanceMonitoring,
+                .performanceMonitoringCoreData,
+                .performanceMonitoringFileIO,
+                .performanceMonitoringNetworking,
+                .performanceMonitoringViewController,
+                .performanceMonitoringUserInteraction:
+            return buildConfig != .appStore
```

Comment:
In case we decide to land this PR before we have support for remote feature toggles: Do not collect metrics in production.

This seems counterintuitive because production is exactly where we'd like to monitor performance. But, I think we ought to first see this in action in a test environment.